src/main/webapp/WEB-INF/tags/instructor/courseStudentDetailsEdit/studentInformationTable.tag

```diff
@@ -4,11 +4,14 @@
 <%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
 <%@ attribute name="studentInfoTable" type="teammates.ui.template.StudentInfoTable" required="true" %>
 <%@ attribute name="newEmail" required="true" %>
+<%@ attribute name="openOrPublishedEmailSentForTheCourse" required="true" %>
 <%@ tag import="teammates.common.util.Const" %>
 <div class="panel panel-primary">
     <div class="panel-body fill-plain">
         <form action="<%=Const.ActionURIs.INSTRUCTOR_COURSE_STUDENT_DETAILS_EDIT_SAVE%>" method="post" id="instructor-student-edit-form" class="form form-horizontal">
             <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="${studentInfoTable.course}">
+            <input type="hidden" name="<%=Const.ParamsNames.SESSION_SUMMARY_EMAIL_SEND_CHECK %>" value="false">
+            <input type="hidden" id="openorpublishedemailsent" name="openorpublishedemailsent" value="${openOrPublishedEmailSentForTheCourse}">
```

Comment:
`openorpublishedemailsent` has been defined in `Const.ParamsNames`. Re-use that constant instead.